Content.Client/GameObjects/Components/HUD/Hotbar/HotbarComponent.cs

```diff
@@ -0,0 +1,257 @@
+﻿using System.Collections.Generic;
+using Content.Client.GameObjects.EntitySystems;
+using Content.Client.UserInterface;
+using Content.Shared.GameObjects.Components.HUD.Hotbar;
+using Robust.Client.GameObjects;
+using Robust.Client.UserInterface.Controls;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Input;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.IoC;
+
+namespace Content.Client.GameObjects.Components.HUD.Hotbar
+{
+    [RegisterComponent]
+    public class HotbarComponent : SharedHotbarComponent
+    {
+#pragma warning disable 649
+        [Dependency] private readonly IGameHud _gameHud;
+        [Dependency] private readonly IHotbarManager _hotbarManager;
+#pragma warning restore 649
+
+        private HotbarGui _hotbarGui;
+        private ActionMenu _actionMenu;
+        private HotbarAction _menuActionSelected;
+
+        private List<HotbarAction> _actions;
+        private List<List<HotbarAction>> _hotbars;
+        private List<HotbarAction> _hotbar;
```

Comment:
```suggestion
        private List<HotbarAction> _hotbar => _hotbars[hotbarIndex];
```
though obviously you'd have to rip out all the setters elsewhere in the code.